### ISSUE TITLE
Nearby crash on connection request

### ISSUE THREAD

**Issue by coen22** [playgameservices/play-games-plugin-for-unity#1965]
Unity  I  Starting to discover
                         I  UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
                         I  UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
                         I  UnityEngine.Logger:Log(LogType, Object)
                         I  UnityEngine.Debug:Log(Object)
                         I  NearbyConnectionsClient:Discover(IDiscoveryListener) (at /Users/coenhacking/Dropbox/Cone 
                            Team Folder/Coonquest/Assets/Scripts/NearbyConnectionsClient.cs:65)
                         I  NearbyConnectionsClient:<NearbyConnectionsClient>m__0(INearbyConnectionClient) (at /Users
                            /coenhacking/Dropbox/Cone Team Folder/Coonquest/Assets/Scripts/NearbyConnectionsClient.cs
                            :34)
                         I  GooglePlayGames.<InitializeNearby>c__AnonStorey0:<>m__0(INearbyConnectionClient) (at /Use
                            rs/coenhacking/Dropbox/Cone Team Folder/Coonquest/Assets/GooglePlayGames/ISocialPlatform/
                            PlayGamesPlatform.cs:262)
                         I  GooglePlayGames.Native.NativeNearbyConnectionClientFactory:OnManagerInitialized(Initializ
                            ationStatus) (at /Users/coenhacking/Dropbox/Cone Team Folder/Coonquest/Assets/GooglePlayG
                            ames/Platforms/Native/NativeNearbyConnectionClientFactory.cs:70)
                         I  Google
                         D  Setting up 1 worker threads for Enlighten.
                         D    Thread -> id: ffffffffc7d01920 -> priority: 1
                         I  OnAdvertisingResult: Success
                         I  UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
                         I  UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
                         I  UnityEngine.Logger:Log(LogType, Object)
                         I  UnityEngine.Debug:Log(Object)
                         I  NearbyConnectionsClient:<Advertise>m__1(AdvertisingResult) (at /Users/coenhacking/Dropbox
                            /Cone Team Folder/Coonquest/Assets/Scripts/NearbyConnectionsClient.cs:55)
                         I  GooglePlayGames.Native.PInvoke.<AsOnGameThreadCallback>c__AnonStorey3:<>m__0() (at /Users
                            /coenhacking/Dropbox/Cone Team Folder/Coonquest/Assets/GooglePlayGames/Platforms/Native/P
                            Invoke/Callbacks.cs:217)
                         I  GooglePlayGames.OurUtils.PlayGamesHelperObject:Update() (at /Users/coenhacking/Dropbox/Co
                            ne Team Folder/Coonquest/Assets/GooglePlayGames/OurUtils/PlayGamesHelperObject.cs:135)
                         I  (Filename: /Users/coenhacking/Dropbox/Cone Team Folder/Coonquest/Assets/Scripts/NearbyCon
                            nectionsClient.cs Line: 55)
         GamesNativeSDK  V  StartDiscoveryOperation completed successfully.
                         V  Detaching from JVM on thread nearby_connections
                    art  F  art/runtime/java_vm_ext.cc:470] JNI DETECTED ERROR IN APPLICATION: use of invalid jobject
                             0xf2105400
                         F  art/runtime/java_vm_ext.cc:470]     from void com.google.android.gms.games.nearby.NativeE
                            ndpointDiscoveryListener.onEndpointFound(java.lang.String, java.lang.String, java.lang.St
                            ring)
                         F  art/runtime/java_vm_ext.cc:470] "main" prio=5 tid=1 Runnable
                         F  art/runtime/java_vm_ext.cc:470]   | group="main" sCount=0 dsCount=0 obj=0x755d4268 self=0
                            xf2105400
                         F  art/runtime/java_vm_ext.cc:470]   | sysTid=19070 nice=-10 cgrp=default sched=0/0 handle=0
                            xf4efa534
                         F  art/runtime/java_vm_ext.cc:470]   | state=R schedstat=( 103214899 21196303 235 ) utm=5 st
                            m=5 core=1 HZ=100
                         F  art/runtime/java_vm_ext.cc:470]   | stack=0xff410000-0xff412000 stackSize=8MB
                         F  art/runtime/java_vm_ext.cc:470]   | held mutexes= "mutator lock"(shared held)
                         F  art/runtime/java_vm_ext.cc:470]   native: #00 pc 0037b96d  /system/lib/libart.so (_ZN3art
                            15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_
                            9ArtMethodEPv+128)
                         F  art/runtime/java_vm_ext.cc:470]   native: #01 pc 0035b791  /system/lib/libart.so (_ZNK3ar
                            t6Thread9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMap+304)
                         F  art/runtime/java_vm_ext.cc:470]   native: #02 pc 00253f89  /system/lib/libart.so (_ZN3art
                            9JavaVMExt8JniAbortEPKcS2_+812)
                         F  art/runtime/java_vm_ext.cc:470]   native: #03 pc 0025424d  /system/lib/libart.so (_ZN3art
                            9JavaVMExt9JniAbortFEPKcS2_z+72)
                         F  art/runtime/java_vm_ext.cc:470]   native: #04 pc 0035eea3  /system/lib/libart.so (_ZNK3ar
                            t6Thread13DecodeJObjectEP8_jobject+278)
                         F  art/runtime/java_vm_ext.cc:470]   native: #05 pc 000ccdf5  /system/lib/libart.so (_ZN3art
                            11ScopedCheck13CheckInstanceERNS_18ScopedObjectAccessENS0_12InstanceKindEP8_jobjectb+120)
                         F  art/runtime/java_vm_ext.cc:470]   native: #06 pc 000cc3b1  /system/lib/libart.so (_ZN3art
                            11ScopedCheck22CheckPossibleHeapValueERNS_18ScopedObjectAccessEcNS_12JniValueTypeE+188)
                         F  art/runtime/java_vm_ext.cc:470]   native: #07 pc 000cb6af  /system/lib/libart.so (_ZN3art
                            11ScopedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE+810)
                         F  art/runtime/java_vm_ext.cc:470]   native: #08 pc 000c4535  /system/lib/libart.so (_ZN3art
                            8CheckJNI12IsInstanceOfEP7_JNIEnvP8_jobjectP7_jclass+452)
                         F  art/runtime/java_vm_ext.cc:470]   native: #09 pc 0011af4b  /data/app/com.cone.coonquest-1
                            /lib/arm/libgpg.so (???)
                         F  art/runtime/java_vm_ext.cc:470]   native: #10 pc 0011b679  /data/app/com.cone.coonquest-1
                            /lib/arm/libgpg.so (_ZNK3gpg13JavaReference9TypeCheckEP7_JNIEnv+40)
                         F  art/runtime/java_vm_ext.cc:470]   native: #11 pc 0011b703  /data/app/com.cone.coonquest-1
                            /lib/arm/libgpg.so (_ZN3gpg13JavaReferenceC1EPNS_9JavaClassEP8_jobjectNS0_9OwnershipEb+30
                            )
                         F  art/runtime/java_vm_ext.cc:470]   native: #12 pc 0011b9e1  /data/app/com.cone.coonquest-1
                            /lib/arm/libgpg.so (_ZN3gpg13JavaReference16WrapJNIParameterEPNS_9JavaClassEP8_jobject+16
                            )
                         F  art/runtime/java_vm_ext.cc:470]   native: #13 pc 00118ac7  /data/app/com.cone.coonquest-1
                            /lib/arm/libgpg.so (_ZN3gpg21NativeOnEndpointFoundEP7_JNIEnvP8_jobjectS3_S3_S3_S3_+70)
                         F  art/runtime/java_vm_ext.cc:470]   native: #14 pc 000ae799  /system/lib/libart.so (art_qui
                            ck_generic_jni_trampoline+40)
                         F  art/runtime/java_vm_ext.cc:470]   native: #15 pc 000a9dc1  /system/lib/libart.so (art_qui
                            ck_invoke_stub_internal+64)
                         F  art/runtime/java_vm_ext.cc:470]   native: #16 pc 0043171b  /system/lib/libart.so (art_qui
                            ck_invoke_stub+230)
                         F  art/runtime/java_vm_ext.cc:470]   native: #17 pc 000b15fd  /system/lib/libart.so (_ZN3art
                            9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JValueEPKc+144)
                         F  art/runtime/java_vm_ext.cc:470]   native: #18 pc 00207ba9  /system/lib/libart.so (_ZN3art
                            11interpreter34ArtInterpreterToCompiledCodeBridgeEPNS_6ThreadEPNS_9ArtMethodEPKNS_7DexFil
                            e8CodeItemEPNS_11ShadowFrameEPNS_6JValueE+200)
                         F  art/runtime/java_vm_ext.cc:470]   native: #19 pc 00201cfb  /system/lib/libart.so (_ZN3art
                            11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instr
                            uctionEtPNS_6JValueE+430)
                         F  art/runtime/java_vm_ext.cc:470]   native: #20 pc 0042bb39  /system/lib/libart.so (MterpIn
                            vokeVirtualQuick+316)
                         F  art/runtime/java_vm_ext.cc:470]   native: #21 pc 000a0994  /system/lib/libart.so (Execute
                            MterpImpl+29972)
                         F  art/runtime/java_vm_ext.cc:470]   native: #22 pc 001e4ee7  /system/lib/libart.so (_ZN3art
                            11interpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+
                            282)
                         F  art/runtime/java_vm_ext.cc:470]   native: #23 pc 001e99ab  /system/lib/libart.so (_ZN3art
                            11interpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_1
                            1ShadowFrameEPNS_6JValueE+114)
                         F  art/runtime/java_vm_ext.cc:470]   native: #24 pc 00201ce3  /system/lib/libart.so (_ZN3art
                            11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instr
                            uctionEtPNS_6JValueE+406)
                         F  art/runtime/java_vm_ext.cc:470]   native: #25 pc 0042bb39  /system/lib/libart.so (MterpIn
                            vokeVirtualQuick+316)
                         F  art/runtime/java_vm_ext.cc:470]   native: #26 pc 000a0994  /system/lib/libart.so (Execute
                            MterpImpl+29972)
                         F  art/runtime/java_vm_ext.cc:470]   native: #27 pc 001e4ee7  /system/lib/libart.so (_ZN3art
                            11interpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+
                            282)
                         F  art/runtime/java_vm_ext.cc:470]   native: #28 pc 001e99ab  /system/lib/libart.so (_ZN3art
                            11interpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_1
                            1ShadowFrameEPNS_6JValueE+114)
                         F  art/runtime/java_vm_ext.cc:470]   native: #29 pc 00201ce3  /system/lib/libart.so (_ZN3art
                            11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instr
                            uctionEtPNS_6JValueE+406)
                         F  art/runtime/java_vm_ext.cc:470]   native: #30 pc 00429e37  /system/lib/libart.so (MterpIn
                            vokeInterface+862)
                         F  art/runtime/java_vm_ext.cc:470]   native: #31 pc 0009ce14  /system/lib/libart.so (Execute
                            MterpImpl+14740)
                         F  art/runtime/java_vm_ext.cc:470]   native: #32 pc 001e4ee7  /system/lib/libart.so (_ZN3art
                            11interpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+
                            282)
                         F  art/runtime/java_vm_ext.cc:470]   native: #33 pc 001e99ab  /system/lib/libart.so (_ZN3art
                            11interpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_1
                            1ShadowFrameEPNS_6JValueE+114)
                         F  art/runtime/java_vm_ext.cc:470]   native: #34 pc 00201ce3  /system/lib/libart.so (_ZN3art
                            11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instr
                            uctionEtPNS_6JValueE+406)
                         F  art/runtime/java_vm_ext.cc:470]   native: #35 pc 0042bb39  /system/lib/libart.so (MterpIn
                            vokeVirtualQuick+316)
                         F  art/runtime/java_vm_ext.cc:470]   native: #36 pc 000a0994  /system/lib/libart.so (Execute
                            MterpImpl+29972)
                         F  art/runtime/java_vm_ext.cc:470]   native: #37 pc 001e4ee7  /system/lib/libart.so (_ZN3art
                            11interpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+
                            282)
                         F  art/runtime/java_vm_ext.cc:470]   native: #38 pc 001e9911  /system/lib/libart.so (_ZN3art
                            11interpreter30EnterInterpreterFromEntryPointEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sh
                            adowFrameE+92)
                         F  art/runtime/java_vm_ext.cc:470]   native: #39 pc 00420605  /system/lib/libart.so (artQuic
                            kToInterpreterBridge+740)
                         F  art/runtime/java_vm_ext.cc:470]   native: #40 pc 000ae813  /system/lib/libart.so (art_qui
                            ck_to_interpreter_bridge+34)
                         F  art/runtime/java_vm_ext.cc:470]   native: #41 pc 00078705  /system/framework/arm/boot-fra
                            mework.oat (???)
                         F  art/runtime/java_vm_ext.cc:470]   at com.google.android.gms.games.nearby.NativeEndpointDi
                            scoveryListener.onEndpointFound(Native method)
                         F  art/runtime/java_vm_ext.cc:470]   at com.google.android.gms.internal.zzawb$zzd$1.zza(unav
                            ailable:-1)
                         F  art/runtime/java_vm_ext.cc:470]   at com.google.android.gms.internal.zzawb$zzd$1.zzs(unav
                            ailable:-1)
                         F  art/runtime/java_vm_ext.cc:470]   at com.google.android.gms.internal.zzabh.zzb(unavailabl
                            e:-1)
                         F  art/runtime/java_vm_ext.cc:470]   at com.google.android.gms.internal.zzabh$zza.handleMess
                            age(unavailable:-1)
                         F  art/runtime/java_vm_ext.cc:470]   at android.os.Handler.dispatchMessage(Handler.java:102)
                         F  art/runtime/java_vm_ext.cc:470]   at android.os.Looper.loop(Looper.java:154)
                         F  art/runtime/java_vm_ext.cc:470]   at android.app.ActivityThread.main(ActivityThread.java:
                            6334)
                         F  art/runtime/java_vm_ext.cc:470]   at java.lang.reflect.Method.invoke!(Native method)
                         F  art/runtime/java_vm_ext.cc:470]   at com.android.internal.os.ZygoteInit$MethodAndArgsCall
                            er.run(ZygoteInit.java:886)
                         F  art/runtime/java_vm_ext.cc:470]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.j
                            ava:776)
                         F  art/runtime/java_vm_ext.cc:470]
                         F  art/runtime/runtime.cc:422] Runtime aborting...
                         F  art/runtime/runtime.cc:422] Aborting thread:
                         F  art/runtime/runtime.cc:422] "main" prio=10 tid=1 Native
                         F  art/runtime/runtime.cc:422]   | group="" sCount=0 dsCount=0 obj=0x755d4268 self=0xf210540
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19070 nice=-10 cgrp=default sched=0/0 handle=0xf4e
                            fa534
                         F  art/runtime/runtime.cc:422]   | state=R schedstat=( 184839533 22600733 342 ) utm=11 stm=7
                             core=0 HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xff410000-0xff412000 stackSize=8MB
                         F  art/runtime/runtime.cc:422]   | held mutexes= "abort lock"
                         F  art/runtime/runtime.cc:422]   native: #00 pc 0037b96d  /system/lib/libart.so (_ZN3art15Du
                            mpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_9Art
                            MethodEPv+128)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 0035b791  /system/lib/libart.so (_ZNK3art6Th
                            read9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMap+304)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 0034e29d  /system/lib/libart.so (_ZNK3art10A
                            bortState10DumpThreadERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEPNS_6ThreadE+24)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 0034e119  /system/lib/libart.so (_ZNK3art10A
                            bortState4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+424)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 00344ad5  /system/lib/libart.so (_ZN3art7Run
                            time5AbortEPKc+92)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 000b716f  /system/lib/libart.so (_ZN3art10Lo
                            gMessageD2Ev+966)
                         F  art/runtime/runtime.cc:422]   native: #06 pc 00254127  /system/lib/libart.so (_ZN3art9Jav
                            aVMExt8JniAbortEPKcS2_+1226)
                         F  art/runtime/runtime.cc:422]   native: #07 pc 0025424d  /system/lib/libart.so (_ZN3art9Jav
                            aVMExt9JniAbortFEPKcS2_z+72)
                         F  art/runtime/runtime.cc:422]   native: #08 pc 0035eea3  /system/lib/libart.so (_ZNK3art6Th
                            read13DecodeJObjectEP8_jobject+278)
                         F  art/runtime/runtime.cc:422]   native: #09 pc 000ccdf5  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck13CheckInstanceERNS_18ScopedObjectAccessENS0_12InstanceKindEP8_jobjectb+120)
                         F  art/runtime/runtime.cc:422]   native: #10 pc 000cc3b1  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck22CheckPossibleHeapValueERNS_18ScopedObjectAccessEcNS_12JniValueTypeE+188)
                         F  art/runtime/runtime.cc:422]   native: #11 pc 000cb6af  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE+810)
                         F  art/runtime/runtime.cc:422]   native: #12 pc 000c4535  /system/lib/libart.so (_ZN3art8Che
                            ckJNI12IsInstanceOfEP7_JNIEnvP8_jobjectP7_jclass+452)
                         F  art/runtime/runtime.cc:422]   native: #13 pc 0011af4b  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (???)
                         F  art/runtime/runtime.cc:422]   native: #14 pc 0011b679  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZNK3gpg13JavaReference9TypeCheckEP7_JNIEnv+40)
                         F  art/runtime/runtime.cc:422]   native: #15 pc 0011b703  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg13JavaReferenceC1EPNS_9JavaClassEP8_jobjectNS0_9OwnershipEb+30)
                         F  art/runtime/runtime.cc:422]   native: #16 pc 0011b9e1  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg13JavaReference16WrapJNIParameterEPNS_9JavaClassEP8_jobject+16)
                         F  art/runtime/runtime.cc:422]   native: #17 pc 00118ac7  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg21NativeOnEndpointFoundEP7_JNIEnvP8_jobjectS3_S3_S3_S3_+70)
                         F  art/runtime/runtime.cc:422]   native: #18 pc 000ae799  /system/lib/libart.so (art_quick_g
                            eneric_jni_trampoline+40)
                         F  art/runtime/runtime.cc:422]   native: #19 pc 000a9dc1  /system/lib/libart.so (art_quick_i
                            nvoke_stub_internal+64)
                         F  art/runtime/runtime.cc:422]   native: #20 pc 0043171b  /system/lib/libart.so (art_quick_i
                            nvoke_stub+230)
                         F  art/runtime/runtime.cc:422]   native: #21 pc 000b15fd  /system/lib/libart.so (_ZN3art9Art
                            Method6InvokeEPNS_6ThreadEPjjPNS_6JValueEPKc+144)
                         F  art/runtime/runtime.cc:422]   native: #22 pc 00207ba9  /system/lib/libart.so (_ZN3art11in
                            terpreter34ArtInterpreterToCompiledCodeBridgeEPNS_6ThreadEPNS_9ArtMethodEPKNS_7DexFile8Co
                            deItemEPNS_11ShadowFrameEPNS_6JValueE+200)
                         F  art/runtime/runtime.cc:422]   native: #23 pc 00201cfb  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+430)
                         F  art/runtime/runtime.cc:422]   native: #24 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #25 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #26 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #27 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #28 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #29 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #30 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #31 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #32 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #33 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #34 pc 00429e37  /system/lib/libart.so (MterpInvoke
                            Interface+862)
                         F  art/runtime/runtime.cc:422]   native: #35 pc 0009ce14  /system/lib/libart.so (ExecuteMter
                            pImpl+14740)
                         F  art/runtime/runtime.cc:422]   native: #36 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #37 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #38 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #39 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #40 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #41 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #42 pc 001e9911  /system/lib/libart.so (_ZN3art11in
                            terpreter30EnterInterpreterFromEntryPointEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Shadow
                            FrameE+92)
                         F  art/runtime/runtime.cc:422]   native: #43 pc 00420605  /system/lib/libart.so (artQuickToI
                            nterpreterBridge+740)
                         F  art/runtime/runtime.cc:422]   native: #44 pc 000ae813  /system/lib/libart.so (art_quick_t
                            o_interpreter_bridge+34)
                         F  art/runtime/runtime.cc:422]   native: #45 pc 00078705  /system/framework/arm/boot-framewo
                            rk.oat (???)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.games.nearby.NativeEndpointDiscov
                            eryListener.onEndpointFound(Native method)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzawb$zzd$1.zza(unavaila
                            ble:-1)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzawb$zzd$1.zzs(unavaila
                            ble:-1)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzabh.zzb(unavailable:-1
                            )
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzabh$zza.handleMessage(
                            unavailable:-1)
                         F  art/runtime/runtime.cc:422]   at android.os.Handler.dispatchMessage(Handler.java:102)
                         F  art/runtime/runtime.cc:422]   at android.os.Looper.loop(Looper.java:154)
                         F  art/runtime/runtime.cc:422]   at android.app.ActivityThread.main(ActivityThread.java:6334
                            )
                         F  art/runtime/runtime.cc:422]   at java.lang.reflect.Method.invoke!(Native method)
                         F  art/runtime/runtime.cc:422]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.r
                            un(ZygoteInit.java:886)
                         F  art/runtime/runtime.cc:422]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:
                            776)
                         F  art/runtime/runtime.cc:422] Dumping all threads without appropriate locks held: thread li
                            st lock mutator lock
                         F  art/runtime/runtime.cc:422] All threads:
                         F  art/runtime/runtime.cc:422] DALVIK THREADS (25):
                         F  art/runtime/runtime.cc:422] "main" prio=10 tid=1 Runnable
                         F  art/runtime/runtime.cc:422]   | group="" sCount=0 dsCount=0 obj=0x755d4268 self=0xf210540
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19070 nice=-10 cgrp=default sched=0/0 handle=0xf4e
                            fa534
                         F  art/runtime/runtime.cc:422]   | state=R schedstat=( 222724014 22622087 344 ) utm=13 stm=9
                             core=6 HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xff410000-0xff412000 stackSize=8MB
                         F  art/runtime/runtime.cc:422]   | held mutexes= "abort lock" "mutator lock"(shared held)
                         F  art/runtime/runtime.cc:422]   native: #00 pc 0037b96d  /system/lib/libart.so (_ZN3art15Du
                            mpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_9Art
                            MethodEPv+128)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 0035b791  /system/lib/libart.so (_ZNK3art6Th
                            read9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMap+304)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 0036dcc9  /system/lib/libart.so (_ZN3art14Du
                            mpCheckpoint3RunEPNS_6ThreadE+608)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 00367d77  /system/lib/libart.so (_ZN3art10Th
                            readList13RunCheckpointEPNS_7ClosureE+330)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 00367a75  /system/lib/libart.so (_ZN3art10Th
                            readList4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEb+584)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 0034e0e5  /system/lib/libart.so (_ZNK3art10A
                            bortState4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+372)
                         F  art/runtime/runtime.cc:422]   native: #06 pc 00344ad5  /system/lib/libart.so (_ZN3art7Run
                            time5AbortEPKc+92)
                         F  art/runtime/runtime.cc:422]   native: #07 pc 000b716f  /system/lib/libart.so (_ZN3art10Lo
                            gMessageD2Ev+966)
                         F  art/runtime/runtime.cc:422]   native: #08 pc 00254127  /system/lib/libart.so (_ZN3art9Jav
                            aVMExt8JniAbortEPKcS2_+1226)
                         F  art/runtime/runtime.cc:422]   native: #09 pc 0025424d  /system/lib/libart.so (_ZN3art9Jav
                            aVMExt9JniAbortFEPKcS2_z+72)
                         F  art/runtime/runtime.cc:422]   native: #10 pc 0035eea3  /system/lib/libart.so (_ZNK3art6Th
                            read13DecodeJObjectEP8_jobject+278)
                         F  art/runtime/runtime.cc:422]   native: #11 pc 000ccdf5  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck13CheckInstanceERNS_18ScopedObjectAccessENS0_12InstanceKindEP8_jobjectb+120)
                         F  art/runtime/runtime.cc:422]   native: #12 pc 000cc3b1  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck22CheckPossibleHeapValueERNS_18ScopedObjectAccessEcNS_12JniValueTypeE+188)
                         F  art/runtime/runtime.cc:422]   native: #13 pc 000cb6af  /system/lib/libart.so (_ZN3art11Sc
                            opedCheck5CheckERNS_18ScopedObjectAccessEbPKcPNS_12JniValueTypeE+810)
                         F  art/runtime/runtime.cc:422]   native: #14 pc 000c4535  /system/lib/libart.so (_ZN3art8Che
                            ckJNI12IsInstanceOfEP7_JNIEnvP8_jobjectP7_jclass+452)
                         F  art/runtime/runtime.cc:422]   native: #15 pc 0011af4b  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (???)
                         F  art/runtime/runtime.cc:422]   native: #16 pc 0011b679  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZNK3gpg13JavaReference9TypeCheckEP7_JNIEnv+40)
                         F  art/runtime/runtime.cc:422]   native: #17 pc 0011b703  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg13JavaReferenceC1EPNS_9JavaClassEP8_jobjectNS0_9OwnershipEb+30)
                         F  art/runtime/runtime.cc:422]   native: #18 pc 0011b9e1  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg13JavaReference16WrapJNIParameterEPNS_9JavaClassEP8_jobject+16)
                         F  art/runtime/runtime.cc:422]   native: #19 pc 00118ac7  /data/app/com.cone.coonquest-1/lib
                            /arm/libgpg.so (_ZN3gpg21NativeOnEndpointFoundEP7_JNIEnvP8_jobjectS3_S3_S3_S3_+70)
                         F  art/runtime/runtime.cc:422]   native: #20 pc 000ae799  /system/lib/libart.so (art_quick_g
                            eneric_jni_trampoline+40)
                         F  art/runtime/runtime.cc:422]   native: #21 pc 000a9dc1  /system/lib/libart.so (art_quick_i
                            nvoke_stub_internal+64)
                         F  art/runtime/runtime.cc:422]   native: #22 pc 0043171b  /system/lib/libart.so (art_quick_i
                            nvoke_stub+230)
                         F  art/runtime/runtime.cc:422]   native: #23 pc 000b15fd  /system/lib/libart.so (_ZN3art9Art
                            Method6InvokeEPNS_6ThreadEPjjPNS_6JValueEPKc+144)
                         F  art/runtime/runtime.cc:422]   native: #24 pc 00207ba9  /system/lib/libart.so (_ZN3art11in
                            terpreter34ArtInterpreterToCompiledCodeBridgeEPNS_6ThreadEPNS_9ArtMethodEPKNS_7DexFile8Co
                            deItemEPNS_11ShadowFrameEPNS_6JValueE+200)
                         F  art/runtime/runtime.cc:422]   native: #25 pc 00201cfb  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+430)
                         F  art/runtime/runtime.cc:422]   native: #26 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #27 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #28 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #29 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #30 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #31 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #32 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #33 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #34 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #35 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #36 pc 00429e37  /system/lib/libart.so (MterpInvoke
                            Interface+862)
                         F  art/runtime/runtime.cc:422]   native: #37 pc 0009ce14  /system/lib/libart.so (ExecuteMter
                            pImpl+14740)
                         F  art/runtime/runtime.cc:422]   native: #38 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #39 pc 001e99ab  /system/lib/libart.so (_ZN3art11in
                            terpreter33ArtInterpreterToInterpreterBridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Sha
                            dowFrameEPNS_6JValueE+114)
                         F  art/runtime/runtime.cc:422]   native: #40 pc 00201ce3  /system/lib/libart.so (_ZN3art11in
                            terpreter6DoCallILb0ELb0EEEbPNS_9ArtMethodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11Instructi
                            onEtPNS_6JValueE+406)
                         F  art/runtime/runtime.cc:422]   native: #41 pc 0042bb39  /system/lib/libart.so (MterpInvoke
                            VirtualQuick+316)
                         F  art/runtime/runtime.cc:422]   native: #42 pc 000a0994  /system/lib/libart.so (ExecuteMter
                            pImpl+29972)
                         F  art/runtime/runtime.cc:422]   native: #43 pc 001e4ee7  /system/lib/libart.so (_ZN3art11in
                            terpreterL7ExecuteEPNS_6ThreadEPKNS_7DexFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  art/runtime/runtime.cc:422]   native: #44 pc 001e9911  /system/lib/libart.so (_ZN3art11in
                            terpreter30EnterInterpreterFromEntryPointEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11Shadow
                            FrameE+92)
                         F  art/runtime/runtime.cc:422]   native: #45 pc 00420605  /system/lib/libart.so (artQuickToI
                            nterpreterBridge+740)
                         F  art/runtime/runtime.cc:422]   native: #46 pc 000ae813  /system/lib/libart.so (art_quick_t
                            o_interpreter_bridge+34)
                         F  art/runtime/runtime.cc:422]   native: #47 pc 00078705  /system/framework/arm/boot-framewo
                            rk.oat (???)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.games.nearby.NativeEndpointDiscov
                            eryListener.onEndpointFound(Native method)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzawb$zzd$1.zza(unavaila
                            ble:-1)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzawb$zzd$1.zzs(unavaila
                            ble:-1)
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzabh.zzb(unavailable:-1
                            )
                         F  art/runtime/runtime.cc:422]   at com.google.android.gms.internal.zzabh$zza.handleMessage(
                            unavailable:-1)
                         F  art/runtime/runtime.cc:422]   at android.os.Handler.dispatchMessage(Handler.java:102)
                         F  art/runtime/runtime.cc:422]   at android.os.Looper.loop(Looper.java:154)
                         F  art/runtime/runtime.cc:422]   at android.app.ActivityThread.main(ActivityThread.java:6334
                            )
                         F  art/runtime/runtime.cc:422]   at java.lang.reflect.Method.invoke!(Native method)
                         F  art/runtime/runtime.cc:422]   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.r
                            un(ZygoteInit.java:886)
                         F  art/runtime/runtime.cc:422]   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:
                            776)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "Jit thread pool worker thread 0" prio=10 tid=2 Native (still
                             starting up)
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x0 self=0xea40c000
                         F  art/runtime/runtime.cc:422]   | sysTid=19075 nice=9 cgrp=default sched=0/0 handle=0xf17f7
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 38522029 7906772 26 ) utm=2 stm=1 cor
                            e=3 HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf16f9000-0xf16fb000 stackSize=1022KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait_queue_me+0xcc/0x124
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait+0xec/0x200
                         F  art/runtime/runtime.cc:422]   kernel: do_futex+0xcc/0x47c
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_futex+0xcc/0x144
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 00017880  /system/lib/libc.so (syscall+28)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 000b8e61  /system/lib/libart.so (_ZN3art17Co
                            nditionVariable16WaitHoldingLocksEPNS_6ThreadE+92)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 0036f017  /system/lib/libart.so (_ZN3art10Th
                            readPool7GetTaskEPNS_6ThreadE+168)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 0036e8c3  /system/lib/libart.so (_ZN3art16Th
                            readPoolWorker3RunEv+62)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 0036e3f1  /system/lib/libart.so (_ZN3art16Th
                            readPoolWorker8CallbackEPv+64)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00047a47  /system/lib/libc.so (_ZL15__pthrea
                            d_startPv+22)
                         F  art/runtime/runtime.cc:422]   native: #06 pc 0001a165  /system/lib/libc.so (__start_threa
                            d+6)
                         F  art/runtime/runtime.cc:422]   (no managed stack frames)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "Signal Catcher" prio=10 tid=3 WaitingInMainSignalCatcherLoop
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d181f0 self=0xe01add0
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19076 nice=0 cgrp=default sched=0/0 handle=0xf16f6
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 488281 0 1 ) utm=0 stm=0 core=3 HZ=10
                            0
                         F  art/runtime/runtime.cc:422]   | stack=0xf15fa000-0xf15fc000 stackSize=1014KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: do_sigtimedwait+0xd8/0x1b4
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_rt_sigtimedwait+0x98/0xd4
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 0004929c  /system/lib/libc.so (__rt_sigtimed
                            wait+12)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 0001e6ef  /system/lib/libc.so (sigwait+34)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 003516c3  /system/lib/libart.so (_ZN3art9Sig
                            nalSet4WaitEv+22)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 003512c3  /system/lib/libart.so (_ZN3art13Si
                            gnalCatcher13WaitForSignalEPNS_6ThreadERNS_9SignalSetE+162)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 00350017  /system/lib/libart.so (_ZN3art13Si
                            gnalCatcher3RunEPv+198)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00047a47  /system/lib/libc.so (_ZL15__pthrea
                            d_startPv+22)
                         F  art/runtime/runtime.cc:422]   native: #06 pc 0001a165  /system/lib/libc.so (__start_threa
                            d+6)
                         F  art/runtime/runtime.cc:422]   (no managed stack frames)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "JDWP" prio=10 tid=4 WaitingInMainDebuggerLoop
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d18280 self=0xea40d40
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19077 nice=0 cgrp=default sched=0/0 handle=0xf15f7
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 919167 30521 2 ) utm=0 stm=0 core=2 H
                            Z=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf14fb000-0xf14fd000 stackSize=1014KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: unix_stream_read_generic+0x24c/0x730
                         F  art/runtime/runtime.cc:422]   kernel: unix_stream_recvmsg+0x34/0x3c
                         F  art/runtime/runtime.cc:422]   kernel: sock_recvmsg+0x44/0x54
                         F  art/runtime/runtime.cc:422]   kernel: ___sys_recvmsg+0xe4/0x1b4
                         F  art/runtime/runtime.cc:422]   kernel: __sys_recvmsg+0x40/0x64
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_recvmsg+0x10/0x18
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 0004a224  /system/lib/libc.so (recvmsg+8)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 00427929  /system/lib/libart.so (_ZN3art4JDW
                            P12JdwpAdbState15ReceiveClientFdEv+88)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 00427b57  /system/lib/libart.so (_ZN3art4JDW
                            P12JdwpAdbState6AcceptEv+354)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 00263d17  /system/lib/libart.so (_ZN3art4JDW
                            P9JdwpState3RunEv+274)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 00263637  /system/lib/libart.so (_ZN3art4JDW
                            PL15StartJdwpThreadEPv+22)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00047a47  /system/lib/libc.so (_ZL15__pthrea
                            d_startPv+22)
                         F  art/runtime/runtime.cc:422]   native: #06 pc 0001a165  /system/lib/libc.so (__start_threa
                            d+6)
                         F  art/runtime/runtime.cc:422]   (no managed stack frames)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "ReferenceQueueDaemon" prio=10 tid=5 Waiting
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d18310 self=0xe01af10
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19078 nice=0 cgrp=default sched=0/0 handle=0xf14f8
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 2005364 25885 14 ) utm=0 stm=0 core=5
                             HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf13f6000-0xf13f8000 stackSize=1038KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait_queue_me+0xcc/0x124
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait+0xec/0x200
                         F  art/runtime/runtime.cc:422]   kernel: do_futex+0xcc/0x47c
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_futex+0xcc/0x144
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 00017880  /system/lib/libc.so (syscall+28)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 000b8e61  /system/lib/libart.so (_ZN3art17Co
                            nditionVariable16WaitHoldingLocksEPNS_6ThreadE+92)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 002bca2d  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadExibNS_11ThreadStateE+516)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 002be263  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+118)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 002cd1df  /system/lib/libart.so (_ZN3artL11O
                            bject_waitEP7_JNIEnvP8_jobject+32)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00000557  /system/framework/arm/boot.oat (Ja
                            va_java_lang_Object_wait__+74)
                         F  art/runtime/runtime.cc:422]   at java.lang.Object.wait!(Native method)
                         F  art/runtime/runtime.cc:422]   - waiting on <0x03dd75c8> (a java.lang.Class<java.lang.ref.
                            ReferenceQueue>)
                         F  art/runtime/runtime.cc:422]   at java.lang.Daemons$ReferenceQueueDaemon.run(Daemons.java:
                            158)
                         F  art/runtime/runtime.cc:422]   - locked <0x03dd75c8> (a java.lang.Class<java.lang.ref.Refe
                            renceQueue>)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.run(Thread.java:761)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "FinalizerDaemon" prio=10 tid=6 Waiting
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d183a0 self=0xe01af60
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19079 nice=0 cgrp=default sched=0/0 handle=0xf13f3
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 5285681 1253905 21 ) utm=0 stm=0 core
                            =6 HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf12f1000-0xf12f3000 stackSize=1038KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait_queue_me+0xcc/0x124
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait+0xec/0x200
                         F  art/runtime/runtime.cc:422]   kernel: do_futex+0xcc/0x47c
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_futex+0xcc/0x144
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 00017880  /system/lib/libc.so (syscall+28)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 000b8e61  /system/lib/libart.so (_ZN3art17Co
                            nditionVariable16WaitHoldingLocksEPNS_6ThreadE+92)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 002bca2d  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadExibNS_11ThreadStateE+516)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 002be263  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+118)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 002cd20b  /system/lib/libart.so (_ZN3artL13O
                            bject_waitJIEP7_JNIEnvP8_jobjectxi+36)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00000655  /system/framework/arm/boot.oat (Ja
                            va_java_lang_Object_wait__JI+96)
                         F  art/runtime/runtime.cc:422]   at java.lang.Object.wait!(Native method)
                         F  art/runtime/runtime.cc:422]   - waiting on <0x0fde4661> (a java.lang.Object)
                         F  art/runtime/runtime.cc:422]   at java.lang.Object.wait(Object.java:407)
                         F  art/runtime/runtime.cc:422]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:
                            188)
                         F  art/runtime/runtime.cc:422]   - locked <0x0fde4661> (a java.lang.Object)
                         F  art/runtime/runtime.cc:422]   at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:
                            209)
                         F  art/runtime/runtime.cc:422]   at java.lang.Daemons$FinalizerDaemon.run(Daemons.java:212)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.run(Thread.java:761)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "FinalizerWatchdogDaemon" prio=10 tid=7 Sleeping
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d18430 self=0xe01afb0
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19080 nice=0 cgrp=default sched=0/0 handle=0xf12ee
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 1376510 150938 8 ) utm=0 stm=0 core=1
                             HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf11ec000-0xf11ee000 stackSize=1038KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait_queue_me+0xcc/0x124
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait+0xec/0x200
                         F  art/runtime/runtime.cc:422]   kernel: do_futex+0xcc/0x47c
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_futex+0xcc/0x144
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 00017884  /system/lib/libc.so (syscall+32)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 000b9289  /system/lib/libart.so (_ZN3art17Co
                            nditionVariable9TimedWaitEPNS_6ThreadExi+108)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 002bca3d  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadExibNS_11ThreadStateE+532)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 002be263  /system/lib/libart.so (_ZN3art7Mon
                            itor4WaitEPNS_6ThreadEPNS_6mirror6ObjectExibNS_11ThreadStateE+118)
                         F  art/runtime/runtime.cc:422]   native: #04 pc 002d0acf  /system/lib/libart.so (_ZN3artL12T
                            hread_sleepEP7_JNIEnvP7_jclassP8_jobjectxi+54)
                         F  art/runtime/runtime.cc:422]   native: #05 pc 00085273  /system/framework/arm/boot.oat (Ja
                            va_java_lang_Thread_sleep__Ljava_lang_Object_2JI+126)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.sleep!(Native method)
                         F  art/runtime/runtime.cc:422]   - sleeping on <0x0b1bb586> (a java.lang.Object)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.sleep(Thread.java:371)
                         F  art/runtime/runtime.cc:422]   - locked <0x0b1bb586> (a java.lang.Object)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.sleep(Thread.java:313)
                         F  art/runtime/runtime.cc:422]   at java.lang.Daemons$FinalizerWatchdogDaemon.sleepFor(Daemo
                            ns.java:327)
                         F  art/runtime/runtime.cc:422]   at java.lang.Daemons$FinalizerWatchdogDaemon.waitForFinaliz
                            ation(Daemons.java:368)
                         F  art/runtime/runtime.cc:422]   at java.lang.Daemons$FinalizerWatchdogDaemon.run(Daemons.ja
                            va:261)
                         F  art/runtime/runtime.cc:422]   at java.lang.Thread.run(Thread.java:761)
                         F  art/runtime/runtime.cc:422]
                         F  art/runtime/runtime.cc:422] "HeapTaskDaemon" prio=10 tid=8 Blocked
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12d184c0 self=0xf2105e0
                            0
                         F  art/runtime/runtime.cc:422]   | sysTid=19081 nice=0 cgrp=default sched=0/0 handle=0xf11e9
                            920
                         F  art/runtime/runtime.cc:422]   | state=S schedstat=( 36745363 1206613 252 ) utm=3 stm=0 co
                            re=2 HZ=100
                         F  art/runtime/runtime.cc:422]   | stack=0xf10e7000-0xf10e9000 stackSize=1038KB
                         F  art/runtime/runtime.cc:422]   | held mutexes=
                         F  art/runtime/runtime.cc:422]   kernel: __switch_to+0x88/0x94
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait_queue_me+0xcc/0x124
                         F  art/runtime/runtime.cc:422]   kernel: futex_wait+0xec/0x200
                         F  art/runtime/runtime.cc:422]   kernel: do_futex+0xcc/0x47c
                         F  art/runtime/runtime.cc:422]   kernel: compat_SyS_futex+0xcc/0x144
                         F  art/runtime/runtime.cc:422]   kernel: el0_svc_naked+0x24/0x28
                         F  art/runtime/runtime.cc:422]   native: #00 pc 00017880  /system/lib/libc.so (syscall+28)
                         F  art/runtime/runtime.cc:422]   native: #01 pc 000b8e61  /system/lib/libart.so (_ZN3art17Co
                            nditionVariable16WaitHoldingLocksEPNS_6ThreadE+92)
                         F  art/runtime/runtime.cc:422]   native: #02 pc 001cd9cd  /system/lib/libart.so (_ZN3art2gc1
                            3TaskProcessor7GetTaskEPNS_6ThreadE+288)
                         F  art/runtime/runtime.cc:422]   native: #03 pc 001ce04b  /system/lib/libart.so (_ZN3art2gc1
                            3TaskProcessor11RunAllTasksEPNS_6ThreadE+42)
                         F  art/runtime/runtime.cc:422]   at java.util.concurrent.ThreadPoolExecutor$Worker.run(Threa
                            dPoolExecutor.java:607)
                         F  art/runtime/runtime.cc:422]   | sysTid=19141 nice=10 cgrp=bg_non_interactive sched=0/0 ha
                            ndle=0xc88eb920
                         F  art/runtime/runtime.cc:422]   at android.net.LocalSocketImpl.-wrap1(LocalSocketImpl.java:
                            -1)
                         F  art/runtime/runtime.cc:422]   | group="" sCount=1 dsCount=0 obj=0x12e53430 self=0xca507b0
                            0
                         F  art/runtime/runtime.cc:422]   - locked <0x076da412> (a com.android.okhttp.ConnectionPool)
                   libc  F  Fatal signal 6 (SIGABRT), code -6 in tid 19070 (.cone.coonquest)
                  DEBUG  F  #00 pc 0004a870  /system/lib/libc.so (tgkill+12)
                         F  #01 pc 00047f73  /system/lib/libc.so (pthread_kill+34)
                         F  #02 pc 0001dd11  /system/lib/libc.so (raise+10)
                         F  #03 pc 00019715  /system/lib/libc.so (__libc_android_abort+34)
                         F  #04 pc 000174b0  /system/lib/libc.so (abort+4)
                         F  #05 pc 00344bc1  /system/lib/libart.so (_ZN3art7Runtime5AbortEPKc+328)
                         F  #06 pc 000b716f  /system/lib/libart.so (_ZN3art10LogMessageD2Ev+966)
                         F  #07 pc 00254127  /system/lib/libart.so (_ZN3art9JavaVMExt8JniAbortEPKcS2_+1226)
                         F  #08 pc 0025424d  /system/lib/libart.so (_ZN3art9JavaVMExt9JniAbortFEPKcS2_z+72)
                         F  #09 pc 0035eea3  /system/lib/libart.so (_ZNK3art6Thread13DecodeJObjectEP8_jobject+278)
                         F  #10 pc 000ccdf5  /system/lib/libart.so (_ZN3art11ScopedCheck13CheckInstanceERNS_18ScopedO
                            bjectAccessENS0_12InstanceKindEP8_jobjectb+120)
                         F  #11 pc 000cc3b1  /system/lib/libart.so (_ZN3art11ScopedCheck22CheckPossibleHeapValueERNS_
                            18ScopedObjectAccessEcNS_12JniValueTypeE+188)
                         F  #12 pc 000cb6af  /system/lib/libart.so (_ZN3art11ScopedCheck5CheckERNS_18ScopedObjectAcce
                            ssEbPKcPNS_12JniValueTypeE+810)
                         F  #13 pc 000c4535  /system/lib/libart.so (_ZN3art8CheckJNI12IsInstanceOfEP7_JNIEnvP8_jobjec
                            tP7_jclass+452)
                         F  #14 pc 0011af4b  /data/app/com.cone.coonquest-1/lib/arm/libgpg.so
                         F  #15 pc 0011b679  /data/app/com.cone.coonquest-1/lib/arm/libgpg.so (_ZNK3gpg13JavaReferenc
                            e9TypeCheckEP7_JNIEnv+40)
                         F  #16 pc 0011b703  /data/app/com.cone.coonquest-1/lib/arm/libgpg.so (_ZN3gpg13JavaReference
                            C1EPNS_9JavaClassEP8_jobjectNS0_9OwnershipEb+30)
                         F  #17 pc 0011b9e1  /data/app/com.cone.coonquest-1/lib/arm/libgpg.so (_ZN3gpg13JavaReference
                            16WrapJNIParameterEPNS_9JavaClassEP8_jobject+16)
                         F  #18 pc 00118ac7  /data/app/com.cone.coonquest-1/lib/arm/libgpg.so (_ZN3gpg21NativeOnEndpo
                            intFoundEP7_JNIEnvP8_jobjectS3_S3_S3_S3_+70)
                         F  #19 pc 000ae799  /system/lib/libart.so (art_quick_generic_jni_trampoline+40)
                         F  #20 pc 000a9dc1  /system/lib/libart.so (art_quick_invoke_stub_internal+64)
                         F  #21 pc 0043171b  /system/lib/libart.so (art_quick_invoke_stub+230)
                         F  #22 pc 000b15fd  /system/lib/libart.so (_ZN3art9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JVal
                            ueEPKc+144)
                         F  #23 pc 00207ba9  /system/lib/libart.so (_ZN3art11interpreter34ArtInterpreterToCompiledCod
                            eBridgeEPNS_6ThreadEPNS_9ArtMethodEPKNS_7DexFile8CodeItemEPNS_11ShadowFrameEPNS_6JValueE+
                            200)
                         F  #24 pc 00201cfb  /system/lib/libart.so (_ZN3art11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMe
                            thodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11InstructionEtPNS_6JValueE+430)
                         F  #25 pc 0042bb39  /system/lib/libart.so (MterpInvokeVirtualQuick+316)
                         F  #26 pc 000a0994  /system/lib/libart.so (ExecuteMterpImpl+29972)
                         F  #27 pc 001e4ee7  /system/lib/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadEPKNS_7D
                            exFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  #28 pc 001e99ab  /system/lib/libart.so (_ZN3art11interpreter33ArtInterpreterToInterpreter
                            BridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11ShadowFrameEPNS_6JValueE+114)
                         F  #29 pc 00201ce3  /system/lib/libart.so (_ZN3art11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMe
                            thodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11InstructionEtPNS_6JValueE+406)
                         F  #30 pc 0042bb39  /system/lib/libart.so (MterpInvokeVirtualQuick+316)
                         F  #31 pc 000a0994  /system/lib/libart.so (ExecuteMterpImpl+29972)
                         F  #32 pc 001e4ee7  /system/lib/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadEPKNS_7D
                            exFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  #33 pc 001e99ab  /system/lib/libart.so (_ZN3art11interpreter33ArtInterpreterToInterpreter
                            BridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11ShadowFrameEPNS_6JValueE+114)
                         F  #34 pc 00201ce3  /system/lib/libart.so (_ZN3art11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMe
                            thodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11InstructionEtPNS_6JValueE+406)
                         F  #35 pc 00429e37  /system/lib/libart.so (MterpInvokeInterface+862)
                         F  #36 pc 0009ce14  /system/lib/libart.so (ExecuteMterpImpl+14740)
                         F  #37 pc 001e4ee7  /system/lib/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadEPKNS_7D
                            exFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  #38 pc 001e99ab  /system/lib/libart.so (_ZN3art11interpreter33ArtInterpreterToInterpreter
                            BridgeEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11ShadowFrameEPNS_6JValueE+114)
                         F  #39 pc 00201ce3  /system/lib/libart.so (_ZN3art11interpreter6DoCallILb0ELb0EEEbPNS_9ArtMe
                            thodEPNS_6ThreadERNS_11ShadowFrameEPKNS_11InstructionEtPNS_6JValueE+406)
                         F  #40 pc 0042bb39  /system/lib/libart.so (MterpInvokeVirtualQuick+316)
                         F  #41 pc 000a0994  /system/lib/libart.so (ExecuteMterpImpl+29972)
                         F  #42 pc 001e4ee7  /system/lib/libart.so (_ZN3art11interpreterL7ExecuteEPNS_6ThreadEPKNS_7D
                            exFile8CodeItemERNS_11ShadowFrameENS_6JValueEb+282)
                         F  #43 pc 001e9911  /system/lib/libart.so (_ZN3art11interpreter30EnterInterpreterFromEntryPo
                            intEPNS_6ThreadEPKNS_7DexFile8CodeItemEPNS_11ShadowFrameE+92)
                         F  #44 pc 00420605  /system/lib/libart.so (artQuickToInterpreterBridge+740)
                         F  #45 pc 000ae813  /system/lib/libart.so (art_quick_to_interpreter_bridge+34)
                         F  #46 pc 016b9705  /system/framework/arm/boot-framework.oat (offset 0x1641000) (android.os.
                            Handler.dispatchMessage+112)
                         F  #47 pc 01c8c297  /system/framework/arm/boot-framework.oat (offset 0x1641000) (android.os.
                            Looper.loop+714)
                         F  #48 pc 016cb81f  /system/framework/arm/boot-framework.oat (offset 0x1641000) (android.app
                            .ActivityThread.main+426)
                         F  #49 pc 000a9dc1  /system/lib/libart.so (art_quick_invoke_stub_internal+64)
                         F  #50 pc 00431823  /system/lib/libart.so (art_quick_invoke_static_stub+226)
                         F  #51 pc 000b1627  /system/lib/libart.so (_ZN3art9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JVal
                            ueEPKc+186)
                         F  #52 pc 0033f3d9  /system/lib/libart.so (_ZN3artL18InvokeWithArgArrayERKNS_33ScopedObjectA
                            ccessAlreadyRunnableEPNS_9ArtMethodEPNS_8ArgArrayEPNS_6JValueEPKc+56)
                         F  #53 pc 00340c17  /system/lib/libart.so (_ZN3art12InvokeMethodERKNS_33ScopedObjectAccessAl
                            readyRunnableEP8_jobjectS4_S4_j+666)
                         F  #54 pc 002da361  /system/lib/libart.so (_ZN3artL13Method_invokeEP7_JNIEnvP8_jobjectS3_S3_
                            +40)
                         F  #55 pc 005afad1  /system/framework/arm/boot.oat (offset 0x51c000) (java.lang.reflect.Meth
                            od.invoke+116)
                         F  #56 pc 0218b329  /system/framework/arm/boot-framework.oat (offset 0x1641000) (com.android
                            .internal.os.ZygoteInit$MethodAndArgsCaller.run+100)
                         F  #57 pc 0218c1af  /system/framework/arm/boot-framework.oat (offset 0x1641000) (com.android
                            .internal.os.ZygoteInit.main+1218)
                         F  #58 pc 000a9dc1  /system/lib/libart.so (art_quick_invoke_stub_internal+64)
                         F  #59 pc 00431823  /system/lib/libart.so (art_quick_invoke_static_stub+226)
                         F  #60 pc 000b1627  /system/lib/libart.so (_ZN3art9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JVal
                            ueEPKc+186)
                         F  #61 pc 0033f3d9  /system/lib/libart.so (_ZN3artL18InvokeWithArgArrayERKNS_33ScopedObjectA
                            ccessAlreadyRunnableEPNS_9ArtMethodEPNS_8ArgArrayEPNS_6JValueEPKc+56)
                         F  #62 pc 0033f037  /system/lib/libart.so (_ZN3art17InvokeWithVarArgsERKNS_33ScopedObjectAcc
                            essAlreadyRunnableEP8_jobjectP10_jmethodIDSt9__va_list+266)
                         F  #63 pc 00294f21  /system/lib/libart.so (_ZN3art3JNI21CallStaticVoidMethodVEP7_JNIEnvP7_jc
                            lassP10_jmethodIDSt9__va_list+424)

                            Process com.cone.coonquest (PID: 19070) ended

**Comment by claywilkinson** [comment]
Thanks @coen22  - This is a known issue in the C++ SDK and should be fixed in an upcoming release.

**Comment by coen22** [comment]
So, if I understand you correctly, I could try a different version of the NDK?

**Comment by claywilkinson** [comment]
The bug is in the libgpg.so which comes from the C++ SDK for Play Game Services.  It is independent of the NDK.  The updated SDK was just released, so I'll be updating this plugin soon.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
